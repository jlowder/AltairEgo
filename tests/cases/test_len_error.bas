10 X = 123
20 PRINT LEN(X)
30 END
