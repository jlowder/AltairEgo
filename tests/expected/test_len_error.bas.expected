Altair Ego: Emulating Altair BASIC 32K Rev. 3.2
OK
OK
TYPE MISMATCH
OK
